ctam/interfaces/fw_update_ifc.py

```diff
@@ -444,7 +454,7 @@ def RedFishFWUpdate(self, BinPath, URI, targets=[], is_multipart=False):
             headers = {"Content-Type": "multipart/form-data"}
             body = {
                 "UpdateFile": (BinPath, open(BinPath, "rb"), "application/octet-stream"),
-                "UpdateParameters" : ("Targets", json.dumps({"Targets": targets, "ForceUpdate": True if self.dut().multipart_force_update else False}),'application/json')
+                "UpdateParameters" : ("Targets", json.dumps({"Targets": targets, "ForceUpdate": True if is_force_update else False}),'application/json')
```

Comment:
The Redfish Firmware Update Whitepaper calls out Targets as a standard parameter. But is ForceUpdate a standard parameter?